.github/workflows/deploy.yml

```diff
@@ -30,13 +30,17 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: 16
-      - name: Install Yalc
-        run: sudo npm i yalc -g
-      - name: NPM Install
-        run: npm ci
+      - name: Checkout amazon-chime-sdk repo
+        uses: actions/checkout@v2
+        with:
+          repository: 'aws-samples/amazon-chime-sdk'
+      - name: Install latest version of React library and JS SDK in meeting demo
+        run: |
+          cd $GITHUB_WORKSPACE/scripts
+          node ./update-sdk-version.js
```

Comment:
Oh - I just noticed something. In your Github Action run -(https://github.com/aws/amazon-chime-sdk-component-library-react/runs/2967692193?check_suite_focus=true)

I see in the `Checkout amazon-chime-sdk` step - 

```
Run actions/checkout@v2
Syncing repository: aws-samples/amazon-chime-sdk
Getting Git version info
/usr/bin/git config --local --get remote.origin.url
https://github.com/aws/amazon-chime-sdk-component-library-react
Deleting the contents of '/home/runner/work/amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react'
Initializing the repository
Disabling automatic garbage collection
Setting up auth
```

The important part here is: `Deleting the contents of '/home/runner/work/amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react'`

It looks like we aren't able to checkout two repos in the same github action like this. Perhaps you can put this update-sdk-version.js script in the amazon-chime-sdk repo instead of the components repo. I think that makes more sense anyways.

Reply:
We need to update sdk version for the release demo when testing anyway so it has to be done here. You can checkout multiple repos using github checkout (See https://github.com/actions/checkout#Checkout-multiple-repos-nested). You just need to specify a path.